### ISSUE TITLE
Fails on start with `IndexError: list index out of range`

### ISSUE THREAD

**Issue by dmakhno** [allure-framework/allure-nose-adaptor#4]
Inherits allure-framework/allure-python#8
Tests were executed in single process.
```
$nosetests --verbose -d --with-allure --logdir=allure-results --with-xunit --xunit-file=report.xml stories
Traceback (most recent call last):
  File "/usr/local/bin/nosetests", line 9, in <module>
    load_entry_point('nose==1.3.4', 'console_scripts', 'nosetests')()
  File "/usr/local/lib/python2.7/dist-packages/nose/core.py", line 121, in __init__
    **extra_args)
  File "/usr/lib/python2.7/unittest/main.py", line 95, in __init__
    self.runTests()
  File "/usr/local/lib/python2.7/dist-packages/nose/core.py", line 207, in runTests
    result = self.testRunner.run(self.test)
  File "/usr/local/lib/python2.7/dist-packages/nose/core.py", line 62, in run
    test(result)
  File "/usr/local/lib/python2.7/dist-packages/nose/suite.py", line 177, in __call__
    return self.run(*arg, **kw)
  File "/usr/local/lib/python2.7/dist-packages/nose/suite.py", line 224, in run
    test(orig)
  File "/usr/local/lib/python2.7/dist-packages/nose/suite.py", line 177, in __call__
    return self.run(*arg, **kw)
  File "/usr/local/lib/python2.7/dist-packages/nose/suite.py", line 224, in run
    test(orig)
  File "/usr/local/lib/python2.7/dist-packages/nose/suite.py", line 177, in __call__
    return self.run(*arg, **kw)
  File "/usr/local/lib/python2.7/dist-packages/nose/suite.py", line 224, in run
    test(orig)
  File "/usr/local/lib/python2.7/dist-packages/nose/suite.py", line 177, in __call__
    return self.run(*arg, **kw)
  File "/usr/local/lib/python2.7/dist-packages/nose/suite.py", line 224, in run
    test(orig)
  File "/usr/local/lib/python2.7/dist-packages/nose/case.py", line 45, in __call__
    return self.run(*arg, **kwarg)
  File "/usr/local/lib/python2.7/dist-packages/nose/case.py", line 138, in run
    result.addError(self, err)
  File "/usr/local/lib/python2.7/dist-packages/nose/proxy.py", line 131, in addError
    plugins.addError(self.test, err)
  File "/usr/local/lib/python2.7/dist-packages/nose/plugins/manager.py", line 99, in __call__
    return self.call(*arg, **kw)
  File "/usr/local/lib/python2.7/dist-packages/nose/plugins/manager.py", line 167, in simple
    result = meth(*arg, **kw)
  File "/usr/local/lib/python2.7/dist-packages/nose_allure/__init__.py", line 70, in addError
    self.allure.impl.stop_case(Status.BROKEN, message=message, trace=trace)
  File "/usr/local/lib/python2.7/dist-packages/allure/common.py", line 161, in stop_case
    test = self.stack[-1]
IndexError: list index out of range
```

**Comment by sergeychipiga** [comment]
Hi, @dmakhno 
Thanks for your report. Seems some kind of magic is around ```stories``` word

**Comment by sergeychipiga** [comment]
@dmakhno, Did I realize correct, that ```stories``` is a name of non-exist folder?

**Comment by dmakhno** [comment]
`stories` is existing module with tests... I just tried to adopt working line with allure plugin. think of me as early adopter.

> And without `--with-allure --logdir=allure-report` It will work that?
it's works fine for a long time this way on ci server

**Comment by jimboliv** [comment]
I can confirm the same behavior.  I pulled down the nose adaptor plugin and tried to use it with tests that have been stable for months.  

```
(.env)[root@atclvmlms20 acceptance]# nosetests ./tests.py
...........
----------------------------------------------------------------------
Ran 11 tests in 16.259s

OK
(.env)[root@atclvmlms20 acceptance]# nosetests --with-allure --logdir=allure ./tests.py
Traceback (most recent call last):
  File "/home/test/.env/bin/nosetests", line 11, in <module>
    sys.exit(run_exit())
  File "/home/test/.env/lib/python2.7/site-packages/nose/core.py", line 121, in __init__
    **extra_args)
  File "/opt/ericsson/enmutils/.env/lib/python2.7/unittest/main.py", line 95, in __init__
    self.runTests()
  File "/home/test/.env/lib/python2.7/site-packages/nose/core.py", line 207, in runTests
    result = self.testRunner.run(self.test)
  File "/home/test/.env/lib/python2.7/site-packages/nose/core.py", line 62, in run
    test(result)
  File "/home/test/.env/lib/python2.7/site-packages/nose/suite.py", line 177, in __call__
    return self.run(*arg, **kw)
  File "/home/test/.env/lib/python2.7/site-packages/nose/suite.py", line 224, in run
    test(orig)
  File "/home/test/.env/lib/python2.7/site-packages/nose/suite.py", line 177, in __call__
    return self.run(*arg, **kw)
  File "/home/test/.env/lib/python2.7/site-packages/nose/suite.py", line 224, in run
    test(orig)
  File "/home/test/.env/lib/python2.7/site-packages/nose/suite.py", line 177, in __call__
    return self.run(*arg, **kw)
  File "/home/test/.env/lib/python2.7/site-packages/nose/suite.py", line 224, in run
    test(orig)
  File "/home/test/.env/lib/python2.7/site-packages/nose/case.py", line 45, in __call__
    return self.run(*arg, **kwarg)
  File "/home/test/.env/lib/python2.7/site-packages/nose/case.py", line 138, in run
    result.addError(self, err)
  File "/home/test/.env/lib/python2.7/site-packages/nose/proxy.py", line 131, in addError
    plugins.addError(self.test, err)
  File "/home/test/.env/lib/python2.7/site-packages/nose/plugins/manager.py", line 99, in __call__
    return self.call(*arg, **kw)
  File "/home/test/.env/lib/python2.7/site-packages/nose/plugins/manager.py", line 167, in simple
    result = meth(*arg, **kw)
  File "/home/test/.env/lib/python2.7/site-packages/nose_allure/__init__.py", line 70, in addError
    self.allure.impl.stop_case(Status.BROKEN, message=message, trace=trace)
  File "/home/test/.env/lib/python2.7/site-packages/allure/common.py", line 161, in stop_case
    test = self.stack[-1]
IndexError: list index out of range
(.env)[root@atclvmlms20 acceptance]# python --version
Python 2.7.8
(.env)[root@atclvmlms20 acceptance]# nosetests --version
nosetests version 1.3.4
(.env)[root@atclvmlms20 acceptance]# pip show nose-allure-plugin
---
Name: nose-allure-plugin
Version: 1.0
Location: /home/test/.env/lib/python2.7/site-packages
Requires: nose, pytest-allure-adaptor
```

**Comment by sergeychipiga** [comment]
@jimboliv thank you for your report! I'm planning to investigate it this weekend, unfortunatelly I was busy in previous

**Comment by sergeychipiga** [comment]
@jimboliv can you pls attach your tests.py file, if it's not secret or too complicated?

**Comment by sergeychipiga** [comment]
@jimboliv seems, I understood the problem. Do you use unittest module, isn't it?

**Comment by sergeychipiga** [comment]
@dmakhno, @jimboliv pls update plugin from pypi. The problem must be fixed

**Comment by jimboliv** [comment]
Sergey,

My tests are unittest-style tests.  I pulled your update and re-ran the tests, and it ran successfully.  I even ran nosetests with two processes and gave it two test modules, and the test modules were executed in parallel successfully.  Thanks for the patch!!

Everything went well, but I did notice one small issue.  The nose.allure.feature, nose.allure.issue and nose.allure.story decorators don't seem to be working correctly as I don't see the features or stories when I click on the Behaviours tab in the report. This might be a known issue.  If it is a new issue, let me know and I can open a new bug tomorrow with more information and test examples.
